models/DataObject/Objectbrick/Definition/Dao.php

```diff
@@ -40,6 +46,21 @@ public function getTableName(DataObject\ClassDefinition $class, $query = false)
         }
     }
 
+    /**
+     * @param DataObject\ClassDefinition $class
+     * @param bool $query
+     * @param string $language
+     *
+     * @return string
+     */
+    public function getLocalizedTableName(DataObject\ClassDefinition $class, $query = false, $language = 'en')
+    {
+        if ($query) {
+            return 'object_brick_localized_query_' . $this->model->getKey() . '_' . $class->getId() . '_' . $language;
+        } else {
+            return 'object_brick_localized_' . $this->model->getKey() . '_' . $class->getId();
```

Comment:
```suggestion
            return 'object_brick_localized_' . $this->model->getKey() . '_' . $class->getId() . '_' . $language;
```

Reply:
Ok. This is correct in the PR. Could you please fix here also?https://github.com/pimcore/pimcore/blob/3a986d76cd918072bef3c056e6eb55fe8b856b80/models/DataObject/Objectbrick/Definition.php#L132 thanks!

Reply:
Pushed directly on 10.2 branch https://github.com/pimcore/pimcore/commit/8818767b22dd64be082831f95f1d6cba5528b6dd